lib/distro_entry.py

```diff
@@ -29,6 +29,8 @@ def GetDistro():
       return "debian"
     if distribution in ["red hat enterprise linux server"]:
       return "redhat"
+    if distribution in ["centos linux"]:
```

Comment:
cool thanks! Do you mind putting the centos string in the list right above since we have it already?